2023_VillaIslas_Science/2023_VillaIslas_Science.fam

```diff
@@ -0,0 +1,27 @@
+Ranas	11R_R_b_MNT	Ranas	0	1	0
+Toluquilla	2417C_TOL_a_MNT	Toluquilla	0	2	0
+Toluquilla	2417J_TOL_a_MNT	Toluquilla	0	1	0
+Toluquilla	2417Q_TOL_b_MNT	Toluquilla	0	1	0
+Toluquilla	333A_TOL_a_MNT	Toluquilla	0	2	0
+Toluquilla	333B_TOL_a_MNT	Toluquilla	0	0	0
+Toluquilla	333C_TOL_a_MNT	Toluquilla	0	2	0
+Toluquilla	333O_TOL_b_MNT	Toluquilla	0	0	0
+Toluquilla	333Q_TOL_b_MNT	Toluquilla	0	0	0
+Ranas	37AI_R_b_MNT	Ranas	0	0	0
+Toluquilla	6428A_TOL_b_MNT	Toluquilla	0	1	0
+Ranas	7A_R_b_MNT	Ranas	0	1	0
+CañadadelaVirgen	E10_CdV_b_MNT	CañadadelaVirgen	0	0	0
```

Comment:
As already discussed elsewhere I think we should avoid non-ASCII characters in individual and primary group IDs.